cmd/tfimport/main.go

```diff
@@ -16,10 +16,11 @@
 // Requires Terraform to be installed and for authentication to be configured for each provider in the Terraform configs provider blocks.
 //
 // Usage:
-// $ go run . [--input_dir=/path/to/config]
+// $ go run . [--input_dir=/path/to/config] [--resource_types 'google_storage_bucket.bucket' --resources 'google_resource_manager_lien.lien']
```

Comment:
no need to include the resource name in the argument, and consider updating the README.md as well

Reply:
Opened https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/783 to fix this.